src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The name 'AreaCalculator' suggests a verb, which is not recommended for class/interface names. Consider using a noun that represents the entity, such as 'AreaComputable' or 'AreaProvider'.